staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -793,6 +794,12 @@ type KubeletConfiguration struct {
 	// Default: false
 	// +optional
 	KernelMemcgNotification bool `json:"kernelMemcgNotification,omitempty"`
+	// Logging specifies the options of logging.
+	// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
+	// Defaults:
+	//   Format: text
```

Comment:
If this says it defaults to `"text"`, I expected to see defaulting in pkg/kubelet/apis/config/v1beta1/defaults.go or vendor/k8s.io/component-base/config/v1alpha1/defaults.go that set that default. That might also let us drop the conditional `if s.Logging.Format != "" {` checks before applying the config format to the log options.